eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -27,8 +27,4 @@ extends:
         Path: sdk/cosmos/cosmos-emulator-matrix.json
         Selection: all
         GenerateVMJobs: true
-    Artifacts:
```

Comment:
@scbedd do we want to make the parameter default value this instead of empty array? Or are there more artifacts? If it's just an empty array it will be hard to override in the runtime UI without knowing the required yaml format.